README.md

```diff
@@ -90,7 +88,7 @@ Install this project with:
 ` cd School-library
   - Run gem install after cloning the repo
 
-### Usage
+
 
 
 
```

Comment:
![image](https://github.com/ridabensalem/blog-app/assets/93039359/0c484855-3d14-4e95-8513-cb6546950503)

- [ ] It would be nice if you fix the highlighted link highlighted in the image above to make your readme more descriptive and professional 👍